palettes.h

```diff
@@ -19,15 +18,24 @@ enum PaletteIndex {
 	PI_HORNS_SHADOW = 7,
 	_PALETTE_SIZE
 };
-class Palette : public Identifiable<std::array<Color, _PALETTE_SIZE>> {
+class PaletteData : public Identifiable<std::array<Color, _PALETTE_SIZE>> {
 public:
-	Palette(const std::array<Color, _PALETTE_SIZE> &colors);
-	Palette(const std::initializer_list<Color> &i_list);
+	PaletteData(const std::array<Color, _PALETTE_SIZE> &colors);
+	PaletteData(const std::initializer_list<Color> &i_list);
+	PaletteData(const std::array<std::string, _PALETTE_SIZE - 1> &hex_strings);
+};
+
+enum class PaletteGroup {
+	All,

```

Comment:
Some enums like this one are really just tags, not stand-ins for things that should have been proper objects, and thus are spared from my crusade.